orttraining/orttraining/python/training/torchdynamo/ort_backend.py

```diff
@@ -486,7 +529,16 @@ def _ort_acclerated_call(self, graph_module: torch.fx.GraphModule, *args, **kwar
             #
             # WARNING: The downstream code should not change prim_outputs and
             # this backend should always produces output with schema identical to prim_outputs'.
-            prim_outputs = FakeTensorProp(graph_module).propagate(*args, **kwargs)
+            try:
+                prim_outputs = FakeTensorProp(graph_module).propagate(*args, **kwargs)
+            except Exception:
+                print(f"FakeTensorProb failed for {graph_module}")
```

Comment:
Let's use logger, e. g., `logger.info("extra_support_dict supports node.target: %s (type: %s)", node.target, type(node.target))`. The uses of logger can be found in this file.